src/coreComponents/mesh/CellElementSubRegion.hpp

```diff
@@ -379,6 +405,12 @@ class CellElementSubRegion : public ElementSubRegionBase
   /// Map from local Cell Elements to Embedded Surfaces
   EmbSurfMapType m_toEmbeddedSurfaces;
 
+  /// List of the elements sharing some face with the interface elements (useful for bubble elements)
+  array1d< localIndex > m_bubbleCells;
+
+  /// List of face ids which are shared with the interface elements (useful for bubble elements)
+  array2d< localIndex > m_toFaceElements;
```

Comment:
I am not sure I fully understand these 2 lists.  They also don't seem to be initialized nor registered as wrappers. I would do that. Make sure you set the sizeFromParent to false. 

it seems to be that `m_bubbleCells` just contains the indices of all 3D cells that are connected to a 2D fracture cell. I would say: 
"List of the elements adjacent to a faceElement (useful for bubble elements)"

The second list is a bit more confusing. What's the size of that array2d? Are you trying to store the `localIndex` of each face connected to the `faceElement` ?  That map already exists in the `FaceElementSubRegion` so I would use that one.